bucket/geogebra5.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "5.0.675.0",
+    "description": "A dynamic mathematics software for education that brings together geometry, algebra, spreadsheets, graphing, statistics and calculus.",
+    "homepage": "https://www.geogebra.org/",
+    "license": {
+        "identifier": "freeware for non-commercial use",
+        "url": "https://www.geogebra.org/license"
+    },
+    "url": "https://download.geogebra.org/installers/5.0/GeoGebra-Windows-Portable-5-0-675-0.zip",
+    "hash": "0d6b8d21ee44e68ddd00337fbf7672af4318ad9fd3f1b88cd5230fd5f79d1163",
+    "installer": {
+      "script": [
+        "$old_dir = (Get-ChildItem ($dir.Path + '\\GeoGebra *')).Fullname",
+        "Move-Item -Path ($old_dir + '\\*') -Destination $dir",
+        "Remove-Item $old_dir"
+      ]
+    },
+    "bin": [
+        [
+            "GeoGebra.exe",
+            "geogebra5"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "GeoGebra.exe",
+            "GeoGebra 5"
+        ]
+    ],
+    "checkver": {
+        "url": "https://download.geogebra.org/installers/5.0/version.txt",
+        "re": "(\\d+)-(\\d+)-(\\d+)-(\\d+)",
+        "replace": "${1}.${2}.${3}.${4}"
+    },
+    "autoupdate": {
+        "url": "https://download.geogebra.org/installers/5.0/GeoGebra-Windows-Portable-$dashVersion.zip"
```

```suggestion
        "url": "https://download.geogebra.org/installers/5.0/GeoGebra-Windows-Portable-$dashVersion.zip",
        "extract_dir": "geogebra $version"
```
Use `extract_dir` instead. This is clearer, and easier to maintain.

---

Oops. Should be `geogebra 5.0.675.0`. I didn't put it correctly.
Autoupdate for `extract_dir` will not work if `extract_dir` does not exist in the manifest.

---

No worries, you're totally right. Now it's much simpler without the installer script! :D